make test_e2e_node: can't find several packages

Well, the first thing I ran into was:

```
~/go-tools/src/k8s.io/kubernetes • make test_e2e_node                        11:53  mtaufen@mtaufen0
hack/e2e-node-test.sh FOCUS= SKIP=
You do not appear to have ginkgo built. Try 'hack/build-go.sh github.com/onsi/ginkgo/ginkgo'
make: *** [test_e2e_node] Error 1
```

`hack/build-go.sh github.com/onsi/ginkgo/ginkgo` failed, presumably because things have moved to `vendor/`?

I had success with this:
```
~/go-tools/src/k8s.io/kubernetes • hack/build-go.sh vendor/github.com/onsi/ginkgo/ginkgo
Go version: go version go1.6.2 linux/amd64
+++ [0516 11:56:19] Building go targets for linux/amd64:
    vendor/github.com/onsi/ginkgo/ginkgo
+++ [0516 11:56:19] Placing binaries
```

And now I'm stuck on this:

```
~/go-tools/src/k8s.io/kubernetes • make test_e2e_node                        11:56  mtaufen@mtaufen0
hack/e2e-node-test.sh FOCUS= SKIP=
Failed to compile e2e_node:

../../../../../../../../../../test/e2e_node/e2e_build.go:28:2: cannot find package "github.com/golang/glog" in any of:
	HOME/go/src/github.com/golang/glog (from $GOROOT)
	HOME/go-tools/src/k8s.io/kubernetes/_output/local/go/src/github.com/golang/glog (from $GOPATH)
../../../../../../../../../../test/e2e_node/container.go:28:2: cannot find package "github.com/onsi/gomega/format" in any of:
	HOME/go/src/github.com/onsi/gomega/format (from $GOROOT)
	HOME/go-tools/src/k8s.io/kubernetes/_output/local/go/src/github.com/onsi/gomega/format (from $GOPATH)
../../../../../../../../../../test/e2e_node/container.go:29:2: cannot find package "github.com/onsi/gomega/types" in any of:
	HOME/go/src/github.com/onsi/gomega/types (from $GOROOT)
	HOME/go-tools/src/k8s.io/kubernetes/_output/local/go/src/github.com/onsi/gomega/types (from $GOPATH)

Ginkgo ran 1 suite in 2.846503152s
Test Suite Failed
!!! Error in hack/e2e-node-test.sh:32
  '"${ginkgo}" --focus=$focus --skip=$skip "${KUBE_ROOT}/test/e2e_node/" -- --alsologtostderr --v 2 --node-name $(hostname) --build-services=true --start-services=true --stop-services=true' exited with status 1
Call stack:
  1: hack/e2e-node-test.sh:32 main(...)
Exiting with status 1
make: *** [test_e2e_node] Error 1
```

I replaced my personal paths to home in that error with `HOME` to make things easier to read. Don't interpret that as the script talking about an environment var or anything.

I have the required packages in my `$GOPATH`, but it seems that Kubernetes wants to grab them from it's own custom location. Am I supposed to run some other script that puts the source for these deps in `_output/local/go/src`?

--- a/hack/e2e-node-test.sh
+++ b/hack/e2e-node-test.sh
@@ -16,19 +16,17 @@
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
-kube::golang::setup_env
 
 focus=${FOCUS:-""}
 skip=${SKIP:-""}
 
 ginkgo=$(kube::util::find-binary "ginkgo")
 if [[ -z "${ginkgo}" ]]; then
-  echo "You do not appear to have ginkgo built. Try 'hack/build-go.sh github.com/onsi/ginkgo/ginkgo'"
+  echo "You do not appear to have ginkgo built. Try 'make WHAT=vendor/github.com/onsi/ginkgo/ginkgo'"
   exit 1
 fi
 
 # Provided for backwards compatibility
-sudo -v
 "${ginkgo}" --focus=$focus --skip=$skip "${KUBE_ROOT}/test/e2e_node/" -- --alsologtostderr --v 2 --node-name $(hostname) --build-services=true --start-services=true --stop-services=true
 
 exit $?

